collections/ansible_collections/massopencloud/esi/roles/subnet/defaults/main.yaml

```diff
@@ -0,0 +1,6 @@
+subnet_cidr: 192.168.50.0/24
```

Comment:
We should default this to something larger than /24. I would probably go with a /22.